### PR TITLE
EWB-1798: Support including linked containers in get_equipment_for_container(s)

### PR DESCRIPTION
**Pull request by Lax125**

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [EWB-1798 Update Python SDK](https://app.clickup.com/t/6929263/EWB-1798)

**Comment by Lax125** [comment]
- [ ] Accept https://github.com/zepben/evolve-grpc/pull/58

**Comment by Lax125** [comment]
- [ ] Rebase onto main after https://github.com/zepben/evolve-sdk-python/pull/104 gets merged

File: test/streaming/get/test_network_consumer.py [hunk 1]
```diff
@@ -249,6 +249,48 @@ async def client_test():
...
+                )
+            ).throw_on_error().value
+
+            assert len(mor.objects) == self.service.len_of(Equipment) == 3
+            _assert_contains_mrids(self.service, "fsp", "c2", "tx")
+
+        await self.mock_server.validate(client_test, [StreamGrpc('getEquipmentForContainers', [_create_container_equipment_responses(feeder_network)])])
+
+    @pytest.mark.asyncio
+    async def test_get_equipment_for_containers(self, feeder_network: NetworkService):
```
Comment (charlta) [inline test/streaming/get/test_network_consumer.py:285]: todo: test name has already been used so replaces the above test. Please rename the tests to something appropriate.

**Review by charlta — CHANGES_REQUESTED** [review]

**Review by charlta — APPROVED** [review]

**Pull request merged**